BlueDBOnDisk/src/main/java/io/bluedb/disk/BlueDbOnDisk.java

```diff
@@ -47,5 +72,10 @@ public void shutdown() throws BlueDbException {
 	public Path getPath() {
 		return path;
 	}
-	
+
+	protected List<BlueCollectionOnDisk<?>> getAllCollectionsFromDisk() {
+		List<File> subfolders = FileManager.getFolderContents(path.toFile(), (f) -> f.isDirectory());
+		List<BlueCollectionOnDisk<?>> collections = Blutils.map(subfolders, (File f) -> new BlueCollectionOnDisk<Serializable>(this, f.getName(), Serializable.class) );
+		return collections;
```

Comment:
So, I feel like this could be done using the streams API in a cleaner way.

`List<BlueCollectionOnDisk<?>> collections = subFolders.stream().map(f -> new BlueCollectionOnDisk<Serializable>(this, f.getName()), Serializable.class)).collect(Collectors.toList());`

You could even pull the mapping function out and use it as a method reference. Maybe something like Bluetils::collectionFromFile. 

I just don't like having new code that does something that's already written.

Reply:
How strongly do you feel about that?  In general, I find the streams API to be clunky - it clutters up the code a lot.